### DISCUSSION TITLE
Is there a specific reason why table actions alignment is hard-coded when using grid layout?

### DISCUSSION THREAD

**Discussion by pmartelletti** [filamentphp/filament#8467] [Help]
When trying to use

```php 
$table->actionsAlignment('end')
```
in conjunction with a grid display, I've noticed that the alignment of the actions is forced to be 'start'. Is there a particular reason for this? I'd like to have the actions aligned at the end, but it's currently not possible without having to override this view.

I also understand that the alignment might be intended for the "header" actions, rather than the record actions. However, I think it makes sense to also use them in the record actions when using the grid layout.

Any thoughts?

**Comment by bencagri** [comment]
it seems hardcoded here. https://github.com/filamentphp/tables/blob/4285a031dd36250a86710631a5b1fea1372097a1/resources/views/index.blade.php#L643

indeed, is there any reason? `justify-end` would be a better view.

**Comment by martinArg96** [comment]
I came up with this as a workaround for now:
            
            ->actions([
                Tables\Actions\EditAction::make()->extraAttributes(['style' => 'margin-left: auto;']),

                Action::make('duplicate').....,
                    
                Tables\Actions\DeleteAction::make()-->extraAttributes(['style' => 'margin-right: auto;']),
            ])

**Reply by lourbert** [reply]
![image](https://github.com/user-attachments/assets/92c3b503-d533-4a8d-9a32-eb04bdaab42e)
how to put the button at the end of row, aligned with the other column

**Comment by e-m-s-y** [comment]
This seems to do the trick. It aligns the header actions to the right, which are in the second div of the .fi-header. First div contains the title. Add this to your custom theme:

```
.fi-header div:nth-of-type(2) {
    @apply justify-end;
}
```
